src/Prefix.jl

```diff
@@ -467,15 +517,14 @@ addition to the `true`/`false` signifying whether verification completed
 successfully.
 """
 function verify(path::AbstractString, hash::AbstractString; verbose::Bool = false,
-                report_cache_status::Bool = false)
+                report_cache_status::Bool = false, hash_path::AbstractString="$(path).sha256")
```

Comment:
Look at `install` call to `verify(tarball_path, hash; verbose=verbose, hash_path=hash_path)` in the case where `tarball_path` was changed to `tarball_path = tarball_url`, in which case `hash_path` is different from what `verify` would construct.

As mentioned above, when `tarball_url` is a path rather than a URL, the `.sha256` file was previously written to the directory of the tarball.  This change fixes it to always write the `.sha256` file to the `<prefix>/download` directory.

Reply:
Ah, I see.  Thanks.